features/networking/pod.feature

```diff
@@ -298,3 +298,87 @@ Feature: Pod related networking scenarios
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/aosqe-pod-for-ping.json |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | name=hello-pod |
+    And evaluation of `pod` is stored in the :client_pod clipboard
+    # 'yes' command will send a character "h" continously for 3 seconds to /dev/udp on listener where the node is listening for udp traffic on exposed nodeport. The 3 seconds mechanism will create an Assured
+    #  entry which will give us enough time to validate upcoming steps
+    When I run the :exec background client command with:
+      | pod              | <%= cb.client_pod.name %>                             |
+      | oc_opts_end      |                                                       |
+      | exec_command     | bash                                                  |
+      | exec_command_arg | -c                                                    |
+      | exec_command_arg | yes "h">/dev/udp/<%= cb.node_ip %>/<%= cb.nodeport %> |
+    Given 3 seconds have passed
+    And I terminate last background process
+    
+    #Creating network test pod to levearage conntrack tool
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/net_admin_cap_pod.yaml" replacing paths:
+      | ["spec"]["nodeName"] | <%= cb.nodes[0].name %> |
+    Then the step should succeed
+    Given a pod becomes ready with labels:
+      | name=network-pod |
+    And evaluation of `pod.name` is stored in the :network_pod clipboard
+    And I execute on the pod:
+      | bash | -c | conntrack -L \| grep "<%= cb.nodeport %>" |
+    Then the step should succeed
+    And the output should contain:
+      |<%= cb.host_pod1.ip %>|
+
+    #Deleting the udp listener pod which will trigger a new udp listener pod with new IP
+    Given I ensure "<%= cb.host_pod1.name %>" pod is deleted
+    And a pod becomes ready with labels:
+      | name=udp-pods |
+    And evaluation of `pod` is stored in the :host_pod2 clipboard
+
+    # 'yes' command will send a character "h" continously for 3 seconds to /dev/udp on listener where the node is listening for udp traffic on exposed nodeport. The 3 seconds mechanism will create an Assured
+    #  entry which will give us enough time to validate upcoming steps
+    When I run the :exec background client command with:
+      | pod              | <%= cb.client_pod.name %>                             |
+      | oc_opts_end      |                                                       |
+      | exec_command     | bash                                                  |
+      | exec_command_arg | -c                                                    |
+      | exec_command_arg | yes "h">/dev/udp/<%= cb.node_ip %>/<%= cb.nodeport %> |
+    Given 3 seconds have passed
```

Comment:
@anuragthehatter can you pls comment on why the 3 seconds?

Reply:
@pruan-rht Sure. I already mentioned a comment there. Normally if we send 1 char as a traffic, the conntrack table creates entry for just 30 seconds. So running yes command for 3 seconds is kinda we are making sure we are sending enough data to create ASSURED entry in table for 180 seconds as that will give us enough time to validate further steps. 30 seconds is too risky

Reply:
ok, as long as it works, but kind of hackish...should probably want to look for a more reliable alternative...but ok for now.

Reply:
yea, this is same as OCP-19810 which have been running fine so far on 3.11 with this logic. So i implemented same for 4.x with extra 4.x requirements. We could have used ncat tool here and on 3.11 case for more reliability but that expects interactive inputs or complicated automation. I can see in future to better implement ncat. Yes is basic unix/linux command comes inbuilt with all kernels so taking just leverage of it.